databricks/conftest.py

```diff
@@ -15,8 +15,17 @@
 #
 import pytest
 import numpy
+import tempfile
+import atexit
+import shutil
+import uuid
 
 from databricks import koalas
+from databricks.koalas import utils
+
+
+# Initialize Spark session that should be used in doctests or unittests.
+session = utils.default_session({"spark.jars.packages": "io.delta:delta-core_2.12:0.1.0"})
```

Comment:
`_2.11` instead of `_2.12` for Spark 2.4.3 since Spark 2.4.3 is compiled with Scala 2.11 by default?

Reply:
As for Spark < 2.4.2, it might not work properly according to the doc https://github.com/delta-io/delta#compatibility-with-apache-spark-versions.

Reply:
ohh let me fix it soon.